loadbalancer/algorithm_test.go

```diff
@@ -335,10 +336,59 @@ func TestConsistentHashKey(t *testing.T) {
 	}
 
 	for i, ep := range endpoints {
-		key := ep // key equal to endpoint has the same hash and therefore selects it
+		key := fmt.Sprintf("%s-%d", ep, 1) // key equal to endpoint has the same hash and therefore selects it
```

Comment:
I guess here any of [0, hashesPerEndpoint) would work, not only 1

Reply:
Yeah, had this comment updated locally actually but never pushed :sweat_smile: